geode-core/src/main/java/org/apache/geode/internal/InternalDataSerializer.java

```diff
@@ -3615,16 +3593,18 @@ public int getDSFID() {
     }
 
     @Override
-    public void toData(DataOutput out) throws IOException {
-      super.toData(out);
+    public void toData(DataOutput out,
+        SerializationContext context) throws IOException {
+      super.toData(out, context);
```

Comment:
why are we calling `super` here (and in `fromData()`? Both of those methods are no-ops

Reply:
I'm going to leave that alone - there are probably lots of instances where that's being done in geode-core.  Personally I would always invoke the superclass method like this is doing because I shouldn't assume that it's empty and will always stay empty.